### ISSUE TITLE
Bluetooth sharing crash the app on Android TV

### ISSUE THREAD

**Issue by Luca1991** [TeamAmaze/AmazeFileManager#681]
The app will crash while trying to share any file using bluetooth on Android TV. Here is the logcat:
```
07-25 01:26:54.340 9466-9466/com.amaze.filemanager E/AndroidRuntime: FATAL EXCEPTION: main
                                                                     Process: com.amaze.filemanager, PID: 9466
                                                                     android.content.ActivityNotFoundException: Unable to find explicit activity class {com.android.bluetooth/com.android.bluetooth.opp.BluetoothOppLauncherActivity}; have you declared this activity in your AndroidManifest.xml?
                                                                         at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:1805)
                                                                         at android.app.Instrumentation.execStartActivity(Instrumentation.java:1523)
                                                                         at android.app.Activity.startActivityForResult(Activity.java:4224)
                                                                         at android.support.v4.app.BaseFragmentActivityJB.startActivityForResult(BaseFragmentActivityJB.java:50)
                                                                         at android.support.v4.app.FragmentActivity.startActivityForResult(FragmentActivity.java:79)
                                                                         at android.app.Activity.startActivityForResult(Activity.java:4183)
                                                                         at android.support.v4.app.FragmentActivity.startActivityForResult(FragmentActivity.java:859)
                                                                         at android.app.Activity.startActivity(Activity.java:4507)
                                                                         at android.app.Activity.startActivity(Activity.java:4475)
                                                                         at com.amaze.filemanager.utils.share.ShareAdapter$ViewHolder$1.onClick(ShareAdapter.java:77)
                                                                         at android.view.View.performClick(View.java:5609)
                                                                         at android.view.View$PerformClick.run(View.java:22259)
                                                                         at android.os.Handler.handleCallback(Handler.java:751)
                                                                         at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                         at android.os.Looper.loop(Looper.java:154)
                                                                         at android.app.ActivityThread.main(ActivityThread.java:6120)
                                                                         at java.lang.reflect.Method.invoke(Native Method)
                                                                         at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:865)
                                                                         at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
```

I'm working on a fix, and hopefully I'll send a PR soon.

**Comment by Luca1991** [comment]
Okay this might be a device specific issue (NVIDIA Shield TV) because I'm not able to share files using bluetooth even using other methods. I'll open a thread on NVIDIA forum asking if the bluetooth is locked and can be used only with joypads and hearphones.

**Comment by sorryusernameisalreadytaken** [comment]
THX for this report. Is there a chance to disable this function on foster devices or nvidia shieldtv devices?

**Comment by VishnuSanal** [comment]
Dupe #3830, fixed with #3838
